discovery-provider/src/solana/audius_data_transaction_handlers.py

```diff
@@ -277,6 +279,32 @@ def handle_manage_entity(
         # TODO update stems, remixes, challenge
         records.append(track)
         db_models["tracks"][track_id].append(track)
+    elif isinstance(management_action, management_action.Update) and isinstance(
```

Comment:
is there a check on whether the submitting user is indeed the owner of this track?

Reply:
added that here but commented out because we intended on fetching existing records upstream.

https://github.com/AudiusProject/audius-protocol/pull/3014/files#diff-c796608d55e8c57e9d05d3bb9b07ab8a0812da7661471a031fcec85d4a4b7f74R291